gtk-3.22/gtk-widgets.css

```diff
@@ -1771,25 +1772,51 @@ scrollbar.overlay-indicator slider:hover {
     border-radius: 20px;
 }
 
+scrollbar.bottom trough {
+    border-top-width: 1px;
+}
+
+scrollbar.top trough {
+    border-bottom-width: 1px;
+}
+
+scrollbar.left trough {
+    border-right-width: 1px;
+}
+
+scrollbar.right trough {
+    border-left-width: 1px;
+}
+
 scrollbar trough {
     padding: 0;
+    border-style: solid;
+    border-color: alpha (#000, 0.05);
```

Comment:
Can we remove this please? I want to keep the trough flat intentionally

Reply:
Hm. If non-overlay scrollbars will have background looking like part of content... will not junction also better to be transparent?

Reply:
ah... when it is transparent, it has same background, as hovered scrollbar. However, it could be set to base color.